exampleService/Android.bp

```diff
@@ -58,6 +58,10 @@ aidl_interface {
         },
     },
     versions_with_info: [
+        // {
+        //     version: "1",
+        //     imports: [],
+        // },
         // {
```

Comment:
Nu este o practica buna sa avem cod comentat in repository. Codul comentat poate produce confuzii: de ce e comentat? ce face? mai functioneaza? Daca vrei sa salvezi cod, care nu este necesar pentru rularea proiectului(e.g. cod de logging), recomandarea mea este sa il salvezi intr-un branch.